tests/tr1/tests/atomic/test.cpp

```diff
@@ -222,6 +222,72 @@ static void test_atomic_flag() { // test properties of atomic_flag
     CHECK_INT(value, false);
 }
 
+#if _HAS_CXX20

```

Comment:
I think I'd prefer to add a new test to `std`. There's a lot of archaic cruft in `tr1` that we'll ideally get rid of some day by modernizing all of the tests and migrating them into `std`, adding to `tr1` seems counterproductive.

Reply:
reverted this test, will create better test